src/PublicKey.h

```diff
@@ -41,7 +41,7 @@ class PublicKey {
             return PublicKeyType::ed25519;
         case 2:
         case 3:
-            return PublicKeyType::secp256k1;
+            return PublicKeyType::nist256p1;
```

Comment:
It's safe to change this line? @alejandro-isaza

Reply:
No, this change is for my unit test code. and this problem may need you to help solve. As I mentioned last week, `secp256k1` & `nist256p1` 's public key tag are same. we can't verify signature in `nist256p1` now.

Reply:
I will revert this change, sorry.